spot_tools/src/spot_executor/spot_executor.py

```diff
@@ -90,7 +91,20 @@ def execute_gaze(self, command, feedback, pick_next=False):
     def execute_pick(self, command, feedback):
         feedback.print("INFO", "Executing `pick` command")
 
-        success = object_grasp(self.spot_interface, command.object_class, feedback)
+        if command.object_class == "":
```

Comment:
I will try to finish passing through the actual class either today or tomorrow. But if you want to get this PR merged, feel free to leave this in and add an issue mentioning that we need to remove this